common/crypto/openssl/crl.c

```diff
@@ -14,10 +14,15 @@
 #include <string.h>
 #include <time.h>
 
-#include "../magic.h"
 #include "asn1.h"
 #include "crl.h"
 
+#if !defined(OE_BUILD_ENCLAVE)
```

Comment:
The change was suggested by @dthaler. I changed back to the original implementation which is simpler as suggested.

Reply:
@dthaler I found the comment and I think it's predicated on the assumption that the ifdef goes away entirely instead of getting dispersed through all the source files, is that correct? I'm very much of the opinion that build switch handling should obey the DRY principle. If the concern is not wanting host/enclave specific definitions as part of a common file, then it's still possible for the common/magic.h to do the build check once and #include the appropriate host/enclave copy of magic.h (i.e. common/magic.h becomes a redirect header, which is not uncommon in libraries like libc that need to aggregate across different arch implementations, for example).